src/connector/src/source/base.rs

```diff
@@ -162,7 +162,6 @@ pub struct SourceEnumeratorInfo {
 pub struct SourceContext {
     pub actor_id: u32,
     pub source_id: TableId,
-    // There should be a 1-1 mapping between `source_id` & `fragment_id`
```

Comment:
I think this is wrong. 1 `source_id` can correspond to multiple MV's `SourceExecutor`s, and thus multiple `fragment_id`s